salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
You suggestion results in the same error in our environment.
I guess the issue occurs as we have many windows servers in which the software could use a different encoding.
Only latin1 works for me...